pom.xml

```diff
@@ -263,7 +262,6 @@
         <ojdbc.version>12.1.0.2</ojdbc.version>
         <dbunit.version>2.5.4</dbunit.version>
         <spring.test.dbunit.version>1.3.0</spring.test.dbunit.version>
-        <bouncycastle.version>1.59</bouncycastle.version>
```

Comment:
- [ ] Should you remove dependency `bcprov-jdk15on`?
I think when activate dependency `bcprov-jdk15on`, cause error lacked version definition.

Reply:
All definitions of bcprov-jdk15on have been deleted
38ce245ee2f4f94e2773b32fcb3e3630c3cd93ab